infrastructure/iac/types.go

```diff
@@ -40,3 +40,17 @@ type iacIssue struct {
 	Documentation  lsp.Uri        `json:"documentation"`
 	IacDescription iacDescription `json:"iacDescription"`
 }
+
+type IssueData struct {
+	Id            string   `json:"id"`
+	Title         string   `json:"title"`
+	Severity      string   `json:"severity"`
+	PublicId      string   `json:"publicId"`
+	Documentation string   `json:"documentation"`
+	LineNumber    int      `json:"lineNumber"`
+	Issue         string   `json:"issue"`
```

Comment:
What does this represent?

Reply:
This is the [description of the issue](https://github.com/snyk/snyk-intellij-plugin/blob/e20b59ccffac957d4b523351574c2b978d847362/src/main/kotlin/snyk/iac/IacSuggestionDescriptionPanel.kt#L78).

e.g. From [SNYK-CC-K8S-13](https://snyk.io/security-rules/SNYK-CC-K8S-13), it would be the `Issue` description: `The pod is sharing host's IPC namespace`